### PR TITLE
feat: support sfn load balance

### PR DESCRIPTION
**Pull request by wujunzhuo**
This change is [Reviewable](https://reviewable.io/reviews/yomorun/yomo/282)

### CODE REVIEW

File: core/frame/meta_frame.go [hunk 1]
```diff
@@ -1,43 +1,62 @@
...
 
-// MetaFrame is a Y3 encoded bytes, SeqID is a fixed value of TYPE_ID_TRANSACTION.
-// used for describes metadata for a DataFrame.
+const (
+	// LoadBalanceRandomPick means Zipper will choose SFN instance by a load balance strategy.
+	LoadBalanceRandomPick byte = 0x00
+	// LoadBalanceBindInstance means data will be sent to the specific SFN according to instanceID.
+	LoadBalanceBindInstance byte = 0x01
+	// LoadBalanceBroadcast means data will be sent to all SFN instances.
+	LoadBalanceBroadcast byte = 0x02
```
Comment (fanweixiao) [inline core/frame/meta_frame.go:15]: which scenario will be used in for this algorithm?

Comment (wujunzhuo) [reply core/frame/meta_frame.go:15]: https://github.com/xileteam/yomo-use-case-socks5/pull/9/files#diff-746b2b7fb72c2ec38c6ded7abe807bdf4ed6b1c08f09851efbb406c3691a4175R47

File: core/frame/meta_frame.go [hunk 2]
```diff
@@ -1,43 +1,62 @@
...
 	"github.com/yomorun/y3"
 )
 
-// MetaFrame is a Y3 encoded bytes, SeqID is a fixed value of TYPE_ID_TRANSACTION.
-// used for describes metadata for a DataFrame.
+const (
+	// LoadBalanceRandomPick means Zipper will choose SFN instance by a load balance strategy.
+	LoadBalanceRandomPick byte = 0x00
+	// LoadBalanceBindInstance means data will be sent to the specific SFN according to instanceID.
+	LoadBalanceBindInstance byte = 0x01
```
Comment (fanweixiao) [inline core/frame/meta_frame.go:13]: naming can reference this: https://www.cloudflare.com/learning/performance/types-of-load-balancing-algorithms/

File: core/frame/meta_frame.go (hunk 1 above)
Comment (fanweixiao) [reply core/frame/meta_frame.go:15]: emm, why zipper need boradcast request to all the instances?

Comment (wujunzhuo) [reply core/frame/meta_frame.go:15]: to mimic sticky session...

File: core/frame/meta_frame.go (hunk 2 above)
Comment (wujunzhuo) [reply core/frame/meta_frame.go:13]: these are specific means, not the target

**Pull request closed without merging**